bin/improver-generate-topography-bands-mask

```diff
@@ -48,12 +48,12 @@ iris.FUTURE.netcdf_promote = True
 #           within the band are set to 1.
 #   Sea points are masked and set to the default fill_value
 
-THRESHOLDS_DICT = {'land': {'bounds': [[-500., 0.], [0., 50.],
-                                       [50., 100.], [100., 150.],
-                                       [150., 200.], [200., 250.],
-                                       [250., 300.], [300., 400.],
-                                       [400., 500.], [500., 600.]],
-                            'units': 'm'}}
+THRESHOLDS_DICT = {'any_surface_type': {'bounds': [[-500., 0.], [0., 50.],
+                                                   [50., 100.], [100., 150.],
+                                                   [150., 200.], [200., 250.],
+                                                   [250., 300.], [300., 400.],
+                                                   [400., 500.], [500., 600.]],
+                                        'units': 'm'}}
 
 
 def main():
```

Comment:
Please update the default "THRESHOLDS_FILEPATH" argument "help" string - it still says "land".  Also: given the hard-coded key values in the ancillary generation function, I think it would be worth defining somewhere in this CLI the permitted key values for THRESHOLDS_DICT.  Probably also in this "help" string.

Reply:
I've updated the comment. "Land" is the only type of any significance, you could make the key Stephen and it will just assume all surfaces should be included. Only if it finds Land and a land mask, or gets a land mask and the default dictionary is unchanged will it assume differentiate between land and sea.